scripts/lib/libvirt/fixtures/cloud-admin-uefi.xml

```diff
@@ -56,4 +56,8 @@
     </memballoon>
 
   </devices>
+  <qemu:commandline>
+   <qemu:arg value='-global'/>
+   <qemu:arg value='driver=cfi.pflash01,property=secure,value=off'/>
```

Comment:
But we enable it when we create the ovmf packages: https://build.suse.de/package/view_file/SUSE:SLE-12-SP2:Update/ovmf/ovmf.spec?expand=1